src/text.ts

```diff
@@ -99,7 +122,16 @@ export const requestPhoneCode = async ({
     }
 
     await PhoneCode.create({ phone, code })
-    await sendText({ body, to: phone, logger })
+
+    
+    if (yamlConfig.sms_provider === "Twilio") {
```

Comment:
I think the capitalization can be issue here. one will have to remember that this is "Twilio" and not "twilio" and "SMSala". maybe only lower case + `toLowerCase()` would avoid falling into this potential pitfall.